src/components/checkbox/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { ChangeEvent, FC, ReactNode } from 'react';
+import { Checkbox as ChakraCheckbox } from '@chakra-ui/react';
+
+interface Props {
+  name: string;
+  value?: string;
+  isDisabled?: boolean;
+  isChecked?: boolean;
+  onChange?: (event: ChangeEvent<HTMLInputElement>) => void;
+  renderLabel?: ReactNode;
```

Comment:
would it be possible to only accept string for label? Because if I pass `<div>my fantastic label</div>` it would result in invalid HTML since div is a block level element and not allowed inside `label`

Reply:
Yes,I think I can change the prop type to string for this checkbox case. In the future when we face a case where the label needs an icon we can readjust the component changing the prop type, or adding a new one for the icon.